components/src/core/Drawer/DrawerHeader.tsx

```diff
@@ -118,7 +118,14 @@ export type DrawerHeaderProps = TouchableWithoutFeedbackProps & {
     /**
      * Color to use for header text elements
      *
-     * Default: theme.colors.onSurface
+     * @default: theme.colors.onSurface
+     */
+    /**
+     * Color to use for text elements
+     *
+     * @default: Icon: theme.colors.onSurface
+     * Title: theme.colors.primary
+     * Subtitle: theme.colors.onSurfaceVariant
```

Comment:
what's happening here?

Reply:
For Icon default [fontColor](theme.colors.onSurface https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/components/src/core/Drawer/DrawerHeader.tsx#L202) 
For Title default [fontColor](theme.colors.primary https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/components/src/core/Drawer/DrawerHeader.tsx#L66)
For Subtitle default [fontColor](theme.colors.onSurfaceVariant https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/components/src/core/Drawer/DrawerHeader.tsx#L74)

So tried to mention all.

Reply:
Thanks @huayunh !!!